### DISCUSSION TITLE
Patch Release 2.8.5

### DISCUSSION THREAD

**Discussion by APickledWalrus** [SkriptLang/Skript#6633] [General]
# Skript 2.8.5

Skript 2.8.5 is here with some more bug fixes and quality-of-life additions. There is also early support for 1.20.5/1.20.6 (Skript will run and some basic 1.20.5/1.20.6 features will work). Further support will come in the next releases. You can report any issues on our [issue tracker](https://github.com/SkriptLang/Skript/issues).

Happy Skripting!

### Additions
- [#6134](<https://github.com/SkriptLang/Skript/pull/6134>) Added `mushroom cow` alias for `mooshroom`.
- [#6163](<https://github.com/SkriptLang/Skript/pull/6163>) Added tests for vector syntaxes.
- [#6525](<https://github.com/SkriptLang/Skript/pull/6525>) Added support for specifying `charged creeper` rather than `powered creeper`.
- [#6526](<https://github.com/SkriptLang/Skript/pull/6526>) Improved the documentation of the elements expression.
- [#6528](<https://github.com/SkriptLang/Skript/pull/6528>) Improved the documentation of the leash effect.
- [#6628](<https://github.com/SkriptLang/Skript/pull/6628>) The Skript artifact name now includes the plugin version (e.g. `Skript-2.8.5.jar`).
- [#6632](<https://github.com/SkriptLang/Skript/pull/6632>) Added basic support for the new Armadillo and Bogged entities.

### Bug Fixes
- [#6302](<https://github.com/SkriptLang/Skript/pull/6302>) Removed the outdated Location to Chunk converter.
- [#6523](<https://github.com/SkriptLang/Skript/pull/6523>) Fixed an issue that could occur when attempting to spawn non-spawnable entities.
- [#6561](<https://github.com/SkriptLang/Skript/pull/6561>) Fix the move event's example.
- [#6566](<https://github.com/SkriptLang/Skript/pull/6566>) Removed redundant `[the]` in the hotbar expression.
- [#6578](<https://github.com/SkriptLang/Skript/pull/6578>) Fixed an error that could occur with the inventory click event.
- [#6579](<https://github.com/SkriptLang/Skript/pull/6579>) Fixed function parsing issues with ambiguous parameter lists.
- Fixes locations with no world causing an error:
    - [#6590](<https://github.com/SkriptLang/Skript/pull/6590>) Entity look at effect.
    - [#6589](<https://github.com/SkriptLang/Skript/pull/6589>) Blocks (below, above, etc.) expression.
    - [#6588](<https://github.com/SkriptLang/Skript/pull/6588>) Explode effect.
- [#6591](<https://github.com/SkriptLang/Skript/pull/6591>) Fixed an incorrect internal check that determined whether an expression was nullable.
- [#6594](<https://github.com/SkriptLang/Skript/pull/6594>)/[#6604](<https://github.com/SkriptLang/Skript/pull/6594>) Improved the efficiency of element input pattern checks.
- [#6595](<https://github.com/SkriptLang/Skript/pull/6595>) Fixed incorrect coloring with some error messages.
- [#6600](<https://github.com/SkriptLang/Skript/pull/6600>) Fixed an error that could occur when setting the value of a variable.
- [#6619](<https://github.com/SkriptLang/Skript/pull/6619>) Fixed an issue that could occur when reloading a command on newer Paper versions.
- [#6617](<https://github.com/SkriptLang/Skript/pull/6617>)/[#6630](<https://github.com/SkriptLang/Skript/pull/6630>) Fixed multiple issues that could occur when using Skript on 1.20.5/1.20.6.

[Click here to view the full list of commits made since 2.8.4](<https://github.com/SkriptLang/Skript/compare/2.8.4...2.8.5>)

### Java 11
From Skript 2.9.0 onwards, we will be requiring a minimum Java version of 11 to run.

### Help Us Test
We have an [official Discord community](https://discord.gg/ZPsZAg6ygu) for beta testing Skript's new features and releases.

### Thank You

Special thanks to the contributors whose work was included in this version:
- @APickledWalrus
- @Asleeepp
- @Fusezion
- @Moderocky
- @NotSoDelayed
- @Phill310
- @ShaneBeee
- @sovdeeth
- @TheLimeGlass

As always, if you encounter any issues or have some minor suggestions, please report them at https://github.com/SkriptLang/Skript/issues.
If you have any bigger ideas or input for the future of Skript, you can share those too at https://github.com/SkriptLang/Skript/discussions.

<hr /><em>This discussion was created from the release <a href='https://github.com/SkriptLang/Skript/releases/tag/2.8.5'>Patch Release 2.8.5</a>.</em>

**Comment by eeggeegg** [comment]
Happy skripting
